frontend/src/components/VButton.vue

```diff
@@ -10,8 +10,17 @@
       {
         [$style[`${variant}-pressed`]]: isActive,
         [$style[`connection-${connections}`]]: isConnected,
+        [$style[`icon-start-${size}`]]: hasIconStart,
+        [$style[`icon-end-${size}`]]: hasIconEnd,
+        'gap-x-2':
+          (hasIconEnd || hasIconStart) && (size == 'medium' || size == 'large'),
+        'gap-x-1': (hasIconEnd || hasIconStart) && size == 'small',
+        // Custom tailwind classes don't work with CSS modules in Vue.
```

Comment:
```suggestion
        // Custom tailwind classes don't work with CSS modules in Vue so they are written here explicitly instead of accessed off of `$style`.
```

A little extra clarification might help. It look me a few minutes to understand what the comment meant.

Reply:
Thank you for clarification, @zackkrida !